assets/README

```diff
@@ -351,6 +361,19 @@ make all
 
 make install
 
+If you get some compilation errors like:
+error: cannot convert ‘zmq::socket_t’ to ‘void*’ in assignment
+this is due to an incompatible zmq.hpp file (probably provided with your Linux distribution).
+As a work-around, you will need to install cppzmq via:
```

Comment:
Is this still necessary after tango-controls/cppTango#561?

Reply:
@mliszcz @bourtemb No!

Reply:
Good point but I think it is still necessary for this release since we provide cppTango 9.3.3 with it.